_posts/wookiemage/2016-04-28-Colins-Final-Project.md

```diff
@@ -0,0 +1,54 @@
+
+- [ ] Create a function that saves the data the user requests into a dictionary file.
+
+Which is so much more possible. From what we’ve read in the textbook and additionally in Automate the Boring Stuff, this seemed like a good goal to set. Unfortunately, life gets in the way of ambitious plans. I spent nine hours working on this project on Friday last week. I spent a combined total of four more on Saturday and Sunday. I think that’s about all I have to give. Between the end of the semester, with all the other projects and papers due, and my son’s new habit of not sleeping from 4:00 AM to 6:00 AM, I just don’t have more time to devote to this project. I would like to say, though, that I wake up at night thinking about ways to implement this feature. I think I would build it by creating an integer variable that increases every time that a user tells it to store data. I would also create a new string variable that housed the information that the user asks to store. I would do this by creating a base name for the string variable and concatenating the current number from the counter onto it, to create a new variable name. Then, when the user wants to save and export the data, I would have the program open a file, and write each of those strings to the file and then save and close it. I just wish I had the time to work through this. Tempus Fugit.
+
+## Looking ahead
+
+A number of us in this class want to keep going. We’ve made a lot of progress in this class but feel there is still so much we don’t understand. As far as we know, there is no second level python class, so we are looking for other ways to go forward. One such option is INLS 690-224 Visual Analytics; another one is INLS 690-163W Information Analytics. Both these courses rely on a good understanding of programming and will likely help push us further. I’m looking forward to taking these skills that we’ve garnered in this class and applying them to bigger data sets to create good analytics and visualizations.
```

Comment:
Check out the classes that Rob Capra and Jaime Arguello teach- both use or have opportunities to use Python more deeply.  You also can now look outside the department at CS classes like Bioalgortihms that will let you use Python within other domains.